test/dlc_tests/container_tests/bin/pytorch_tests/testPTHVDHelper

```diff
@@ -25,10 +25,12 @@ opener.addheaders = [('User-agent', 'Mozilla/5.0')]
 urllib.request.install_opener(opener)
 import torchvision
 from torchvision import datasets, transforms
+from packaging import version
```

Comment:
nit: Please use `from packaging.version import Version`, as the `Version` class is more intuitive in comparing version strings than `version.parse`.